python/fapolicy_analyzer/util/fs.py

```diff
@@ -3,7 +3,7 @@
 
 
 def stat(path):
-    return subprocess.getoutput(f"stat {path}")
+    return subprocess.getoutput(f"stat \'{path}\'")
```

Comment:
You shouldn't need to escape single quotes inside of doubles in python.

Reply:
It's not a python thing. The shell is interpreting the whole string.

Reply:
Maybe the escapes aren't needed.